source/components/about-us/about-us.css

```
@@ -3,54 +3,51 @@ main {
 }
 
 #introduction-grid {
-  height: 1000px;
-  margin: 100px;
+  height: 800px;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
 }
 #introduction-title-grid {
-  height: 400px;
+  height: 200px;
```

I think the title should be centered within the blue.
Replace with this CSS to fix it:
```suggestion
  line-height: 200px;
```